tools/protoc/versions.bzl

```diff
@@ -0,0 +1,92 @@
+        "protobuf-25.2.zip": "sha384-0Qno40/wqkCHuTM2rCgVd+nS66Ule3YnqsdTGf3bCZKjnQPrQqmufTXyIvttCw7P",
+        "protoc-25.2-linux-aarch_64.zip": "sha384-XoUL9QXarwVOUqdQd6jQQIxWADtJglQKKX6eCn1wqyObKcOMzmBAxIgPs6JZfKzh",
+        "protoc-25.2-linux-ppcle_64.zip": "sha384-uCsYCijlcpLVM5OAoK5AvmZ/ZG4FxFW5nq4YxHhNbM8rkKorhGwePsfdxjWXD0U8",
+        "protoc-25.2-linux-s390_64.zip": "sha384-7jphWhRqYWg8CJLMtCT1F4LZI9b7exag21s4apghBvfrB9qLhv9CfutAElsezfso",
+        "protoc-25.2-linux-x86_32.zip": "sha384-yQzrtHYPcYthqzCvwm7bRFM7Zsjg+x+qju+V8XJKLiyt2b89l3dmTnJuzVlZX1Eo",
+        "protoc-25.2-linux-x86_64.zip": "sha384-nkqix3MGaILzAx76L/MzgH8YtCp0aR7pSL1X18PVHCw8H/KYV6JwP6ox1UBYPFiq",
+        "protoc-25.2-osx-aarch_64.zip": "sha384-YcvtjYGTERd2jJ93Qx1JRMyjCadaIJCqH76yzrLDGQ+Owak1Eu0SEV2hYkwDsr0q",
+        "protoc-25.2-osx-universal_binary.zip": "sha384-xTq7pC3gKeD4FRdW8kVYdwoVpJ8xRiLChLaaOqHWMQrUd7PFGjEn7EooP0FElM7d",
+        "protoc-25.2-osx-x86_64.zip": "sha384-x+WkzgB4xuctw/qwwYOpIJ8iDyz4QobOU60EUZJeTkTjJ4B0E7ag42KNQbTxIXJ2",
+        "protoc-25.2-win32.zip": "sha384-I0dFedVdL5x1X1Wie/IYJDstKZsKZwOZyaeGq1b00xPfoJibc/Jj7yDTBvGKAM8V",
+        "protoc-25.2-win64.zip": "sha384-zwfHd9XCgWXwTNAcC8dUR/9gPKKdF/pbfS2FqRyMzSzaaTfKLkeMRJQSql9CtcFo",
+    },
+    "v25.3": {
+        "protobuf-25.3.tar.gz": "sha384-CORK2b6dJeBIjaE509agfyBXFsQTbNHx/bntTWnz1hgMq3mFQeh5MJXeT2JnGQYG",
+        "protobuf-25.3.zip": "sha384-kDAMVpfn4VNyjywFJiuLoelCfc3AypQMlmfF+VcDWh7pBwp7mZtPdAR29WttBsYn",
+        "protoc-25.3-linux-aarch_64.zip": "sha384-jwCvnInYx9lD4bNy3EI/leVIVGqpyYkJ3qZSXXtIERcaBRIS1hAi4Tluu9Cy4eiy",
+        "protoc-25.3-linux-ppcle_64.zip": "sha384-aLa68jnCzwO7pS0ler84UxwzaL7srl+9fFZuR3ykh4FJIDJW/jCQGTeuos8VfWcI",
+        "protoc-25.3-linux-s390_64.zip": "sha384-PrlMkT15A2RsI4QYbRgvxq/xsNV7i1jzMbIZkSHDUMyZPC2Txf1ydLrmsIdQ/NQP",
+        "protoc-25.3-linux-x86_32.zip": "sha384-GeYiLamJeiA/5/JY1bE9JisEoQwy9T65ZwQHmfc6JFV2VkWbUkRHGJqt3BeCBtP/",
+        "protoc-25.3-linux-x86_64.zip": "sha384-RJ5r9hcTEMpphUBr1LLyYhjTc6k4tGkqGOW/UOHqHCSTGyFKG2DD7hfZ0XHdrc7e",
+        "protoc-25.3-osx-aarch_64.zip": "sha384-B2T/H3pEAm/tEnOLvMhTRae5zq5PCcPMzTjWspe+vjS/CmTxVxnlMH1XjxgibILi",
+        "protoc-25.3-osx-universal_binary.zip": "sha384-mhajph4BOcfzO74CC5u5JJ/eQ/sY6/Js1sDBVTszZmmo0nxqHU065rnDkHM5yy9i",
+        "protoc-25.3-osx-x86_64.zip": "sha384-AUt06iBAyAmj1bB9KVjilbCX3moW/RLLUwJK3XNLzM3JY9/jIcaY61Juf/gCON3A",
+        "protoc-25.3-win32.zip": "sha384-PoVupdbxosMbZs//vJDkLXjTzY2pfF3vI3lJ9J+SoMqsv17r/kFEzH8brM64x0FS",
+        "protoc-25.3-win64.zip": "sha384-Fe23g3o7lyfGIpl8YNK6zLecYvp1A7dRjOozoztLQPoIFYlKmETsjh5+p4USHrkL",
+    },
+    "v26.0-rc3": {
+        "protobuf-26.0-rc3.tar.gz": "sha384-RVFu8PJJCOSXwYTqH7FyWRSgsP1AAjcEa+VViddVTgtd9wYvZjQoQ8jmlFxwfFw+",
+        "protobuf-26.0-rc3.zip": "sha384-JYSXGTSBfwUU6UzqazUTkT3lTZDzx10YdaNQYjojrT7X1Ro1fA+T4tjJw0e8UISV",
```

Comment:
Can we exclude source archives?

Reply:
sure, I was blindly grabbing it all... that's pretty easy